reference/upgrade-guide.md

```diff
@@ -0,0 +1,162 @@
+---
+outline: [1, 2]
+---
+
+# Overview
+
+The Workflow Description Language has a number of **versions** that are
+constantly improving. This guide lists all major versions of the language—the
+major improvements for each version, common pitfalls when upgrading, and how to
+get help.
+
+If you're interested in learning more about how versioning works and how you can
+specify the version of your WDL documents, see the [Versioning section] within
+the Language Guide.
+
+::: tip
+Changes are grouped in terms of their **impact**. The word "impact" here is a
+combination of "how important is the change" along with "how many existing WDL
+documents are likely to be affected by this change". They are subjective
+determinations made by the upgrade guide curator during the curator process, but
+they are often helpful in quickly scanning upgrade elements to see which might
+apply to your situation.
+:::
+
+# WDL v1.2
+
+WDL v1.2 introduces a great deal of new syntax and changes to the specification.
+Notably, changes were backwards compatible with all previous `v1.x` releases.
+This sections only covers the high points related to upgrading—the full release
+notes can be found [here][wdl-v1.2-release-notes].
+
+### Checklist
+
+Use this checklist to ensure you hit all of the sections.
+
+#### Moderate impact changes 
+
+<input type="checkbox" /> New `Directory` type ([link](#new-directory-type)).<br />
+<input type="checkbox" /> Replacement of `runtime` section with `requirements` and `hints` sections ([link](#replacement-of-runtime-with-requirements-and-hints)). <br />
```

Comment:
I would change this to softer language such as "introduction of requirements and hints and deprecation of runtime"